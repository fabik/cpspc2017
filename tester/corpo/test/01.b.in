100
21 24 30 72 42 36 85 54 26 74 63 82 96 58 78 14 76 94 97 63 75 62 50 92 21 69 86 81 38 49 6 3 42 93 88 49 67 77 23 54 26 48 49 33 29 91 80 1 55 33 16 36 36 27 54 51 22 17 93 15 32 64 18 87 1 82 44 44 19 20 71 17 33 25 5 6 91 1 95 57 23 88 3 46 64 78 4 55 25 36 48 61 85 100 16 93 16 78 19
99 72 97 47 40 14 56 67 87 71 30 84 29 34 75 95 8 22 6 44 38 20 18 76 53 1 19 92 7 86 64 28 75 67 63 49 90 25 18 7 28 28 76 37 38 36 41 52 9 40 4 11 4 60 15 70 8 31 99 98 48 60 94 14 45 13 41 5 23 5 45 93 25 26 43 32 65 73 87 56 72 79 39 18 50 75 95 73 57 36 86 77 48 72 52 27 49 82 27
