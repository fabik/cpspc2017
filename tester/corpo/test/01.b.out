99 0 0 23 35 18 0 0 0 0 0 0 0 1 32 0 0 26 29 25 0 25 13 0 21 26 90 0 16 0 0 2 0 0 0 14 0 10 0 0 0 0 0 1 0 0 0 7 89 20 0 0 0 0 0 0 36 0 0 0 0 0 33 4 0 0 10 0 0 0 35 45 0 0 39 20 0 0 0 0 0 21 0 0 0 0 0 0 0 0 0 14 2 6 2 0 53 0 0 0 
